napari/utils/transforms/transforms.py

```diff
@@ -55,9 +55,9 @@ def inverse(self) -> 'Transform':
                 trans._('Inverse function was not provided.', deferred=True)
             )
 
-    def compose(self, transform: 'Transform') -> 'Transform':
+    def compose(self, transform: Optional['Transform']) -> 'Transform':
```

Comment:
I don't love changing the signature of this method because other classes override it and also need to deal with `None` now. I'd prefer to make changes to `_interaction_box_mouse_bindings.py` to handle the case when the input is `None`, as I would expect an error if I compose `None` with an existing transform. What do you think?

Reply:
I implement simplest solution. If you better understand these files feel free to implement fixed version

Reply:
I made a couple of line suggestions to show what I meant.

Reply:
```suggestion
    def compose(self, transform: 'Transform') -> 'Transform':
```